h2o-core/src/test/java/water/TestUtil.java

```diff
@@ -30,6 +32,11 @@
 
 @Ignore("Support for tests, but no actual tests here")
 public class TestUtil extends Iced {
+  {
```

Comment:
IntelliJ Idea enables assertions by default for JUnit tests: https://github.com/JetBrains/intellij-community/blob/306d705e1829bd3c74afc2489bfb7ed59d686b84/resources/src/idea/RunManager.xml#L23

I don't this is a good idea because in a multi-node setup you could end up with only some nodes with enabled assertions (assuming you are running in an environment that doesn't enable assertions by default for you).